bokehjs/src/coffee/renderer/glyph/line.coffee

```diff
@@ -9,7 +9,7 @@ class LineView extends Glyph.View
   _render: (ctx, indices) ->
     drawing = false
     @visuals.line.set_value(ctx)
-
+    
```

Comment:
It took whitespace?  What OS are you on?  Normally I can simply save a file to trigger a build.